support-frontend/assets/pages/contributions-landing/components/StripeCardForm/StripeCardForm.jsx

```diff
@@ -247,21 +275,29 @@ const CardForm = (props: PropTypes) => {
     });
   };
 
-  const setupOneOffHandlers = (): void => {
+  const setupOneOffRecaptcha = (): void => {
     if (window.guardian.recaptchaEnabled) {
       if (window.grecaptcha && window.grecaptcha.render) {
         setupRecaptchaTokenForOneOff();
       } else {
         window.v2OnloadCallback = setupRecaptchaTokenForOneOff;
       }
     }
+  };
 
+  const setupOneOffHandlers = (): void => {
```

Comment:
We've pulled out `setupOneOffRecaptcha` (and `setupRecurringRecaptcha`) because we now need `setupOneOffHandlers` (and `setupRecurringHandlers`) to run everytime the `zipCode` updates (it gets access to `zipCode` in its closure, so if we only run it once at initialisation, we always have `zipCode = ''`